init_venv.sh

```diff
@@ -125,7 +120,15 @@ pip install -r requirements/nncf_compression.txt || exit 1
 echo "Build NNCF extensions ..."
 python -c "import nncf"
 
-pip install -e $SC_SDK_REPO/src/ote_sdk || exit 1
+if ! [[ -z $OTE_SDK_PATH ]]; then
+  pip install -e $OTE_SDK_PATH || exit 1
+elif ! [[ -z $SC_SDK_REPO ]]; then
+  pip install -e $SC_SDK_REPO/src/ote_sdk || exit 1
+else
```

Comment:
Minor recommendation
```suggestion
if [[ ! -z $OTE_SDK_PATH ]]; then
  pip install -e $OTE_SDK_PATH || exit 1
elif [[ ! -z $SC_SDK_REPO ]]; then
  pip install -e $SC_SDK_REPO/src/ote_sdk || exit 1
else
```
The reason: the bash `[[ ... ]]` allows this trick, and (AFAIK) now it is recommended to do so, since handling of `[[...]]` brackets is a part of bash language.
(Whereas the classic POSIX shell `[...]` is just an alias for a separate application `test` that makes tests and logic operations and returns a status code)